python/dask_cudf/dask_cudf/io/tests/test_parquet.py

```diff
@@ -254,9 +254,11 @@ def test_roundtrip_from_dask_partitioned(tmpdir, parts, daskcudf, metadata):
     df_read = dd.read_parquet(tmpdir, engine="pyarrow")
     gdf_read = dask_cudf.read_parquet(tmpdir)
 
+    columns = list(df_read.columns)
+    assert set(df_read.columns) == set(gdf_read.columns)
     dd.assert_eq(
-        df_read.compute(scheduler=dask.get),
-        gdf_read.compute(scheduler=dask.get),
+        df_read.compute(scheduler=dask.get)[columns],
+        gdf_read.compute(scheduler=dask.get)[columns],
```

Comment:
Can we add a `#TODO` comment here with a note that we should remove this once a fix is in?